config/templates/keepalived-template.yaml

```diff
@@ -25,6 +25,35 @@
         automountServiceAccountToken: false
         enableServiceLinks: false
         shareProcessNamespace: true
+        initContainers:
+        - name: config-setup
+          image: {{ .Misc.image }}
+          imagePullPolicy: Always
+          command:
+          - bash
+          - -c
+          - /usr/local/bin/notify.sh
+          env:
+          - name: file
+            value: /etc/keepalived.d/src/keepalived.conf   
+          - name: dst_file
+            value: /etc/keepalived.d/dst/keepalived.conf
+          - name: reachip
+          {{- if .KeepalivedGroup.Spec.InterfaceFromIP }}
+            value: {{ .KeepalivedGroup.Spec.InterfaceFromIP }}
+          {{- else }}
+            value: ""
+          {{- end }}
+          - name: setup
```

Comment:
why is this true here and false in the init-container? What happens if the desired target IP changes in the CR?